templates/deployment.yaml

```diff
@@ -51,6 +51,8 @@ spec:
           - --enable-leader-election
           - --zap-log-level={{ default 4 .Values.log_level }}
           - --force-tenant-prefix={{ default false .Values.force_tenant_prefix }}
+          - --protected-namespace-regex={{ default "" .Values.protected_namespace_regex }}
```

Comment:
Should these parameters follow camel case convention?

Reply:
They should, but you can't use `-` in variable names in helm :(